fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisParser.g4

```diff
@@ -119,6 +119,12 @@ sortItem
     :  expression ordering = (ASC | DESC)?
     ;
 
+limitClause
+    : (LIMIT limit=INTEGER_VALUE)?
+    | (LIMIT limit=INTEGER_VALUE OFFSET offset=INTEGER_VALUE)?
+    | (LIMIT offset=INTEGER_VALUE COMMA limit=INTEGER_VALUE)?
+    ;
```

Comment:
why not set sortClause and limitClause to optional?
```antlr4
queryOrganization
    : sortClause? limitClause?
    ;

sortItem
    :  expression ordering = (ASC | DESC)
    ;

limitClause
    : (LIMIT limit=INTEGER_VALUE)
    | (LIMIT limit=INTEGER_VALUE OFFSET offset=INTEGER_VALUE)
    | (LIMIT offset=INTEGER_VALUE COMMA limit=INTEGER_VALUE)
    ;
```

Reply:
It does not save null-checking efforts, if we make sortClause and LimitClause optional. At least we need to check null for sortClause and limitClause. 
That part of code was updated now. We only check present of limit and offset, not LIMIT and OFFSET.

Reply:
this change is not for save null-checking efforts, the point is:
1. don't use Long.MAX_VALUE to implicitly represents missing limitClause
2. if limit isn't exist, we shouldn't create LogicalLimit